src/encoded/static/components/item-pages/WorkflowRunTracingView.js

```diff
@@ -212,12 +212,6 @@ export class FileViewGraphSection extends WorkflowGraphSection {
         };
     }
 
-    componentWillReceiveProps(nextProps){
-        if (this.props.steps !== nextProps.steps){
-            this.setState(checkIfIndirectOrReferenceNodesExist(nextProps.steps));
-        }
-    }
-
```

Comment:
Forgot why this edit was made, maybe will revert until can explore further.

Reply:
Ok yeah this is safe to remove. It would return `anyReferenceFileNodes` & `anyReferenceFileNodes` which we don't want in state any longer.